src/smdba/postgresqlgate.py

```diff
@@ -137,6 +137,8 @@ def estimate(self):
         """
         Estimate the data.
         """
+
+        PG_VERSION = [int(v_el) for v_el in os.popen("psql --version | sed -e 's/.*\s//g'").read().split('.')]
```

Comment:
Arrgghh, sorry. My bad. In Python `UPPERCASE_VARIABLES` are only immutable constants at class-level or module level. In this case you should lowercase it as `pg_version` and move it down there right before it is used in `if/else` part.